sidecar/config/flags.go

```diff
@@ -136,21 +137,25 @@ var Flags = []cli.Flag{
 		Usage:  "List of Eureka server URLs",
 	},
 	cli.StringFlag{
+		Name:   rulesTypeFlag,
+		EnvVar: envVar(rulesTypeFlag),
+		Usage:  fmt.Sprintf("Rules service type (%s, %s)", Amalgam8Type, KubernetesType),
+	},
+	cli.StringFlag{
 		Name:   controllerURLFlag,
 		EnvVar: envVar(controllerURLFlag),
-		Usage:  "URL for Controller service",
+		Usage:  "URL for Rules service",
```

Comment:
The `--controller_XXX` usage strings should explicitly indicate that this is the Amalgam8 Controller, not "Rules service" or "Amalgam8 Rules".